services/security-headers/security-headers.service.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+const { BaseService } = require('..')
+
+const documentation = `
+<p>
+  The <a href="https://securityheaders.com/">Security Headers</a>
+  provide a easy mechanism to analyze HTTP response headers and give information on how to deploy missing headers.
+</p>
+</p>
+  The scan result will be hidden from the public result list and follow redirects will be on too.
+<p>
+`
+
+module.exports = class SecurityHeaders extends BaseService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'security-headers',
+      pattern: ':protocol(https|http)/:host',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Security Headers',
+        namedParams: {
+          protocol: 'https',
+          host: 'shields.io',
+        },
+        staticPreview: this.render({
+          grade: 'A+',
+        }),
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'securityheaders',
```

Comment:
I'm wondering whether the badge label should be in two words as well.

Reply:
Switched to "security headers".